scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -256,6 +256,25 @@ <h6>Phenotype terms (HPO):</h6>
       {{ wtf.form_field(form.cytoband_end) }}
     </div>
   </div>
+  <div class="form-group">
```

Comment:
This would look much nicer code-wise if you use a grid system like this: https://getbootstrap.com/docs/4.0/layout/grid/#variable-width-content